csv_download_script.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd 
+import requests
+import collections
+import os
+
+AIRTABLE_API_KEY = os.getenv('AIRTABLE_API_KEY')
+
+headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+url = "https://api.airtable.com/v0/appFfYIupTrVQ0HJx/Rapid%20Review%3A%20Estimates?view=SeroTracker:%20Serosurveys%20Reporting%20Prevalence"
+
+
+wanted_fields = ['Prevalence%20Estimate%20Name', 'Publication%20Date', "Grade%20of%20Estimate%20Scope", "Country", 
```

Comment:
Look at how we append the fields to the airtable get request URL in the existing functionality. Looks a bit odd saving a list with all of these strings. We probably want to do what we already have which is save a list with the normal field names, then when we loop through and add to the URL we put in HTML format. Take a look at the functions here: https://github.com/serotracker/iit-backend/blob/master/app/database_etl/airtable_records_handler/airtable_records_getter.py